rundeckapp/grails-app/views/menu/home.next.gsp

```diff
@@ -63,7 +63,6 @@
   <asset:javascript src="menu/home.js"/>
 
   <!-- VUE CSS MODULES -->
-  <asset:stylesheet href="static/css/components/version-notification.css"/>
```

Comment:
This file isn't being created. In fact the components in the folder version-notification do not seem to be used anywhere.